x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/utils.ts

```diff
@@ -225,3 +225,22 @@ export const getMaxPackageName = (
 
   return `${packageName}-${maxPkgPolicyName + 1}`;
 };
+
+export const getCspmCloudShellDefaultValue = (packageInfo: PackageInfo): string => {
```

Comment:
I would do that on different PR and consider it as improvement, as this change is affecting other features which are outside CIS GCP (this feature)

Reply:
I agree that bigger refactorings should happen separately, but we should be careful with this approach of taking the debt too easily with a promise to ourselves to fix it in the future. The dangers are the following:
- the best case is if we take the refactoring right away in the follow-up PR, but even in this case the problem is that the context of the changes is lost and you need to go back to the initial PR to make sure you don't forget anything
- the worse situation is to create a "refactoring" ticket to deal with it sometime later. These tickets tend to stay in the backlog, plus there is always a situation when you find bugs during the QA cycle, fix them only in one place (in this case in GCP code), and don't fix them in other places where the code is duplicated. As a result, you get once identical code diverged and after some time it's super hard even to see these duplications anymore and unwind what has already been changed in each piece of code

All that to say for me these util functions seem quite straightforward to make one generic out of them right away. But if you see the risk of breaking smth and want to do it in a follow-up PR, let's at least cover `getCspmCloudShellDefaultValue` with unit tests so that the refactoring goes smoother

Reply:
yes, I dont want to risk breaking anything at this point as FF is coming really soon